tools/ossf_best_practices/gold_criteria.md

```diff
@@ -0,0 +1,152 @@
+Check the official [report](https://bestpractices.coreinfrastructure.org/en/projects/29?criteria_level=2) as some questions include additional information that might be relevant to understand the context around the question.
+
+
+# Basics
+> What is the human-readable name of the project?
+
+Node.js
+
+> What is a brief description of the project?
+
+Node.js® is a JavaScript runtime built on Chrome's V8 JavaScript engine
+
+> What is the URL for the project (as a whole)?
+
+https://nodejs.org
+
+> What is the URL for the version control repository (it may be the same as the project URL)?
+
+https://github.com/nodejs/node
+
+## Prerequisites
+
+> The project MUST achieve a silver level badge.
+
+**Unmet**
+
+
+## Project oversight
+
+> The project MUST have a "bus factor" of 2 or more. (URL required)
+
+_Possible answers: Met/Unmet_
+
+> The project MUST have at least two unassociated significant contributors.
+
+_Possible answers: Met/Unmet_
+
+## Other
+
+> The project MUST include a copyright statement in each source file, identifying the copyright holder (e.g., the [project name] contributors).
+
+_Possible answers: Met/Unmet_
+
+> The project MUST include a license statement in each source file. This MAY be done by including the following inside a comment near the beginning of each file: SPDX-License-Identifier: [SPDX license expression for project](https://spdx.dev/ids/#how)
+
+_Possible answers: Met/Unmet_
+
+# Change Control
+
+## Public version-controlled source repository
+
+> The project's source repository MUST use a common distributed version control software (e.g., git or mercurial). 
+
+_Possible answers: Met/Unmet_
+
+> The project MUST clearly identify small tasks that can be performed by new or casual contributors. (URL required)
+
+_Possible answers: Met/Unmet_
```

Comment:
```suggestion
**Met**
```
 The tag "good first issue" is a great example. [see](https://github.com/nodejs/node/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)